packages/workspace/src/workspace/workspace.ts

```diff
@@ -1322,6 +1323,9 @@ export const loadWorkspace = async (
     getValue: async (id: ElemID, env?: string): Promise<Value | undefined> => (
       (await elements(env)).get(id)
     ),
+    getConfigValue: async (id: ElemID): Promise<Value | undefined> => (
```

Comment:
nit: we don't need to write the function types here since it is inferred from the `Workspace` type 
```suggestion
    getConfigValue: async id => (
```